library.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-unused-vars */
+const myLibrary = [];
+
+class Book {
+  constructor(author, title, pages, read) {
+    this.author = author;
+    this.title = title;
+    this.pages = pages;
+    this.read = read;
+  }
+}
+
+const addBookToLibrary = (book) => {
+  if (book.title === '' || book.author === '' || book.pages === '') {
+    // eslint-disable-next-line no-throw-literal
+    alert('input field can not be empty');
```

Comment:
- [ ] Good job handling empty fields, but don't use alerts for that, they block your code, are not styleable, and get blocked by the browser if used too much. Just add a `required` attribute to the form inputs instead or display the warning message in a normal element in the DOM.
https://stackoverflow.com/questions/20359807/why-shouldnt-i-use-alert-in-javascript